test_utils/scripts/update_docs.sh

```diff
@@ -43,3 +45,46 @@ else
     exit $?
 fi
 
+# Adding GitHub pages branch. `git submodule add` checks it
+# out at HEAD.
+GH_PAGES_DIR='ghpages'
+git submodule add -q -b gh-pages \
```

Comment:
That's surprising, can we just clone it into /tmp or something?

Reply:
perhaps. this is just me bringing back the old way of doing it (a few months back in history). I agree we could likely use tmp and not add a submodule

Reply:
I am going to wait on changing this. It has been done like this before with success. I would also write it differently but I am not sure it is worth rewriting this.